docs/customer/dialogs.md

```diff
@@ -1,11 +1,13 @@
 # Dialogs
+
 **Management :material-menu-right: Customer :material-menu-right: [Customer Name] :material-menu-right: Dialogs**
 
-**Dialogs** display the status of currently active calls on the account. Use the **Refresh** button to get up-to-date information. 
+**Dialogs** display the status of current active calls on the account.
 
-!!! note "Ended Status"
-    "Ended" indicates a call is over, but the telephony switches are still tearing down the call, including the Real-time Transport Protocol (RTP) stream and any final signaling. After that, the server gathers data for billing. 
+Use the **Refresh** button to get up-to-date information.
 
+!!! note "Ended Status"
+    "Ended" indicates a call is over, but the telephony switches are still tearing down the call, including the Real-time Transport Protocol (RTP) stream and any final signaling. After that, the server gathers data for billing.
 !!! note "Global Dialogs"
     You can also configure Dialogs for specific Customers in **Global :material-menu-right: Dialogs**, which displays all configured Dialogs.
```

Comment:
If you can rewrite this please, basically this option lets a customer see all of their customer's active call.

Reply:
## Re-written the Global Dialogs

note "Global Dialogs"
    You can also see the status of all the calls for specific Customers in **Global :material-menu-right: Dialogs**.

    Also, you can select and end the calls using **End Calls** and the calls are terminaed using the MI Termination.